hayanime

```diff
@@ -232,8 +232,8 @@ ffmpegdler(){
 hist_handler(){
 	[ -e ~/.hayanihist ] || touch ~/.hayanihist 2>/dev/null		# Supress Incase of the Directory is Unwritable
 	if [ "${1}" = "append" ]; then
-		[ -w ~/.hayanihist ] || die "1" "Hist file is Unwritable, Make sure you have access to write to HOME directory"
-		sed "/${3}/d" ~/.hayanihist > ~/.hayanihist.tmp && mv ~/.hayanihist.tmp ~/.hayanihist
+		[ -w ~/.hayanihist ] || die 1 "Hist file is Unwritable, Make sure you have access to write to HOME directory"
+		printf '%s' "$(sed "/${3}/d" ~/.hayanihist)" > ~/.hayanihist
```

Comment:
Note that it expands the command substitution first before writing/redirecting it directly to the same file